news/fix-sasmodels.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* Refactored code utilizing sasmodels to use the new sasview api.
```

Comment:
let's change this to "changed" as "temporarily removed support for SAS characteristic functions until we can migrate to the new sasview api"